_gsocproposals/2019/proposal_AthenaMTResourceMonitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Multi-thread safe resource monitoring infrastructure for the ATLAS experiment
+layout: gsoc_proposal
+project: ATLAS
+year: 2019
+organization:
+  - UCIrvine
+  - UOslo
```

Comment:
Can you please add this organization to the current year? You can just copy [last year's description](https://github.com/HSF/hsf.github.io/blob/master/_gsocorgs/2018/uoslo.md).

Reply:
Done.